conans/model/profile.py

```diff
@@ -30,12 +30,14 @@ def __repr__(self):
         return self.dumps()
 
     def serialize(self):
-        # TODO: Remove it seems dead
+        def _serialize_tool_requires():
```

Comment:
I think we want just the ``repr(ref)``, serializing to string (can be copied and pasted into other places)